package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "typed-rest-client",
-  "version": "0.16.0",
+  "version": "1.0.0",
```

Comment:
Not sure why this happened.

Reply:
note that 1.0.0 is released with a preview tag.  So, you only get if you do npm install typed-rest-client@preview.